main.py

```diff
@@ -1 +1,107 @@
-x = 50
+import pandas as pd
+import pyodbc
+
+def database_connect(server, database):
+    try:
+        conn = pyodbc.connect(
+        Trusted_Connection='Yes',
+        Driver='{ODBC Driver 17 for SQL Server}',
+        Server=server,
+        Database=database
+        )
+        cursor = conn.cursor()
+        return conn, cursor
+
+    except Exception as e:
+        print(f'Error connecting to the database: {e}')
+        return None, None
+
+def create_dbtable(original_file,cursor, conn):
+    try:
+        # Read the original csv data
+        masterlist = pd.read_csv(original_file, delimiter=';')
+        masterlist['Date'] = pd.to_datetime(masterlist['Date'])
+        #Create table in database
+        cursor.execute('''
```

Comment:
Do not put the string directly, create a variable in the file, or a class variable if you are putting it in a class